deploy/kubernetes/elastic-agent-standalone/elastic-agent-standalone-daemonset-configmap.yaml

```diff
@@ -0,0 +1,231 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: agent-node-datastreams
+  namespace: kube-system
+  labels:
+    k8s-app: elastic-agent
+data:
+  agent.yml: |-
+    outputs:
+      default:
+        type: elasticsearch
+        hosts:
+          - >-
+            ${ES_HOST}
+        username: ${ES_USERNAME}
+        password: ${ES_PASSWORD}
+    agent:
+      monitoring:
+        enabled: true
+        use_output: default
+        logs: true
+        metrics: true
+    providers.kubernetes:
```

Comment:
Does this enables "autodiscover"? (I also see `{kubernetes.container.id}` below).

Reply:
Yeap, this is the first basic version of autodiscover support on Agent (based on https://github.com/elastic/beats/pull/21480). Hints are not supported yet, so imagine it's the equivalent of the old one with the static autodiscover templates.